[CI/CD] Code Coverage failing in CI, blocking merge requests.

Earlier today all ACC-1804 build agents refused to spawn, thus leading to a large CI backlog and blocked merges. Upon investigation, it seems this is due to the VM initialization scripts failing which install the code coverage dependencies. In the short term, I have removed the initialization scripts from the images, which allows the VMS to spawn. This unblocks most of the CI runs but we still have a failed test due to the lack of code coverage dependencies... which is blocking.

I am looking into it with high priority, this issue is just for visibility in case anyone runs into it.

-- Spectre 1 mitigations supported

-- CODE_COVERAGE set - enable code coverage tests

CMake Error at CMakeLists.txt:317 (message):

  LCOV is not found.





-- Configuring incomplete, errors occurred!

See also "/home/oeadmin/workspace/pipelines/Azure-Linux/build/CMakeFiles/CMakeOutput.log".

See also "/home/oeadmin/workspace/pipelines/Azure-Linux/build/CMakeFiles/CMakeError.log".

--- a/.jenkins/pipelines/Azure/Linux/Jenkinsfile
+++ b/.jenkins/pipelines/Azure/Linux/Jenkinsfile
@@ -520,8 +520,7 @@ try{
                 "ACC1804 clang-7 Release LVI FULL Tests":          { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'clang-7', 'Release', ['-DLVI_MITIGATION=ControlFlow', '-DLVI_MITIGATION_SKIP_TESTS=OFF']) },
                 "ACC1804 gcc Debug LVI":                           { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'gcc',     'Debug',   ['-DLVI_MITIGATION=ControlFlow', '-DLVI_MITIGATION_SKIP_TESTS=ON']) },
                 "ACC1804 gcc Release LVI":                         { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'gcc',     'Release', ['-DLVI_MITIGATION=ControlFlow', '-DLVI_MITIGATION_SKIP_TESTS=ON']) },
-                // re-add once https://github.com/openenclave/openenclave/issues/3648 is resolved
-                //"ACC1804 Code Coverage Test" :                     { ACCCodeCoverageTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'clang-7', 'Debug') },
+                "ACC1804 Code Coverage Test" :                     { ACCCodeCoverageTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'clang-7', 'Debug') },
 
                 "ACC1604 clang-7 Release LVI snmalloc":            { ACCTest(AGENTS_LABELS["acc-ubuntu-16.04"], 'clang-7', 'Release', ['-DLVI_MITIGATION=ControlFlow', '-DLVI_MITIGATION_SKIP_TESTS=ON', '-DUSE_SNMALLOC=ON']) },
                 "ACC1804 clang-7 Release LVI FULL Tests snmalloc": { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'clang-7', 'Release', ['-DLVI_MITIGATION=ControlFlow', '-DLVI_MITIGATION_SKIP_TESTS=OFF', '-DUSE_SNMALLOC=ON']) },

